lib/kafka/consumer.rb

```diff
@@ -4,6 +4,7 @@
 require "kafka/offset_manager"
 require "kafka/fetcher"
 require "kafka/pause"
+require "concurrent"
```

Comment:
maybe it can be `require "concurrent/map"` to avoid loading the whole library, only the Map that is the needed one, wdyt?

Reply:
I’d like to get rid of the gem dependency altogether. ruby-kafka should be mostly self-contained.

Reply:
if it isn't needed, removal is a way better solution 😉